internal/wire/relationship/relationship_test.go

```diff
@@ -29,38 +32,49 @@ func init() {
 	tests.InitMongo(cfg)
 }
 
-// prefix is what we are looking to delete after the test.
-const prefix = "RTEST_"
+//==============================================================================
 
-// TestUpsertDelete tests if we can add/remove a relationship to/from the db.
-func TestUpsertDelete(t *testing.T) {
+// setup initializes for each indivdual test.
+func setup(t *testing.T) ([]relationship.Relationship, *db.DB) {
 	tests.ResetLog()
-	defer tests.DisplayLog()
+
+	rels, err := relationshipfix.Get()
```

Comment:
We need them for every test.  So instead of have a bit of code to read them in and tear them down in every test, I just created a setup/teardown function that is called by every test.